### DISCUSSION TITLE
ValueError: transformers.__spec__ is None

### DISCUSSION THREAD

**Discussion by abrh119** [Lightning-AI/pytorch-lightning#12709] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
why does this keep on repeating every time i try to run this? 

`!pip install git+https://github.com/PyTorchLightning/pytorch-lightning
!pip install --upgrade pytorch-lightning
print(pl.__version__)
import pytorch_lightning as pl

from pytorch_lightning.metrics.functional import accuracy, f1, auroc
from pytorch_lightning.callbacks import ModelCheckpoint, EarlyStopping
from pytorch_lightning.loggers import TensorBoardLogger`

**Comment by rohitgr7** [comment]
try updating your `transformers` to latest.
I have seen this issue before and AFAIR they fixed it.
